### PR TITLE
Add SIG Architecture Production Readiness spotlight

### PR DESCRIPTION
**Pull request by fsmunoz**
Spotlight on the Production Readiness subproject of SIG Architecture.

Closes:  #437

### CODE REVIEW

**Comment by fsmunoz** [comment]
Will remove WIP and unhold after some internal review and validation, and open the website PR as well.

**Comment by fsmunoz** [comment]
/retitle Add SIG Architecture Production Readiness spotlight

**Comment by sftim** [comment]
/hold

Ideally, let's align the publication dates with https://github.com/kubernetes/website/pull/43474

File: content/en/blog/2023/sig-architecture-prod-readiness-spotlight.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
+---
+layout: blog
+title: "Spotlight on SIG Architecture: Production Readiness"
+slug: sig-architecture-production-readiness-spotlight-2023
+date: 2023-10-24
```
Comment (sftim) [inline content/en/blog/2023/sig-architecture-prod-readiness-spotlight.md:5]:
```suggestion
date: 2023-11-02
```

**Review by sftim — COMMENTED** [review]
One more thing - aligning with https://github.com/kubernetes/website/pull/43474

**Comment by fsmunoz** [comment]
@sftim I copied the content over, including the date, while keeping the different authorship assignment method. It should be OK.

File: content/en/blog/2023/sig-architecture-prod-readiness-spotlight.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
+---
+layout: blog
+title: "Spotlight on SIG Architecture: Production Readiness"
+slug: sig-architecture-production-readiness-spotlight-2023
+date: 2023-11-02
+canonicalUrl: https://www.k8s.dev/blog/2023/24/05/sig-architecture-prod-readiness-spotlight-2023/
```
Comment (sftim) [inline content/en/blog/2023/sig-architecture-prod-readiness-spotlight.md:6]:
```suggestion
```

**Review by sftim — COMMENTED** [review]
/lgtm
/approve

File: content/en/blog/2023/sig-architecture-prod-readiness-spotlight.md (hunk 2 above)
Comment (sftim) [reply content/en/blog/2023/sig-architecture-prod-readiness-spotlight.md:6]: @fsmunoz this front matter doesn't do anything harmful here, but it'd be tidier if you sent in a follow-up PR to omit it.

Comment (fsmunoz) [reply content/en/blog/2023/sig-architecture-prod-readiness-spotlight.md:6]: Will do! Thanks, forgot about that difference.

**Pull request merged**